### PR TITLE
Add a docs page for the service controller

### PR DESCRIPTION
**Pull request by nckturner**
**What type of PR is this?**
> Uncomment only one, leave it on its own line:

/kind documentation

**What this PR does / why we need it**:
Add documentation for load balancer annotations.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/service_controller.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# The Service Controller
+
+The service controller is responsible for watch for service and node object changes, so that it can create, update, or delete cloud load balancers corresponding to load balanced services.  Like the other controllers, we import the cloud-provider provided utility functions for managing the controller itself, which calls into cloud provider defined methods `GetLoadBalancer`, `GetLoadBalancerName`, `EnsureLoadBalancer`, `UpdateLoadBalancer`, and `EnsureLoadBalancerDeleted`.
+
```
Comment (ayberk) [inline docs/service_controller.md:4]: Annotation values have no constraints, right? I looked at the docs and looks like there are rules only for the keys. Just making sure so we don't give the users the wrong impression with `Valid Values : - `

File: mkdocs.yml [hunk 2]
```diff
@@ -5,6 +5,8 @@ repo_name: kubernetes/cloud-provider-aws/
 repo_url: https://github.com/kubernetes/cloud-provider-aws/
 nav:
   - 'Home': README.md
+  - 'Components':
```
Comment (ayberk) [inline mkdocs.yml:8]: This is minor, but I think these tabs are listed by the order they have here. We might want to move it to between user guide and development.

**Review by ayberk — APPROVED** [review]
Nothing to block this PR. Minor suggestions that can be done later.

File: docs/service_controller.md (hunk 1 above)
Comment (nckturner) [reply docs/service_controller.md:4]: Yeah it might not be quite the correct way to phrase it..

File: mkdocs.yml (hunk 2 above)
Comment (nckturner) [reply mkdocs.yml:8]: That makes sense.  We can do in a follow up.

**Pull request merged**